src/Components/MainSection/TaskList/Task.jsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
+import { useTasklist } from '../../../Context/tasklist-context';
+
+function Task({ task }) {
+	const { tasklistDispatch } = useTasklist();
 
-function Task() {
 	return (
-		<div className="task task-pending">
-			<input
-				className="task-checkbox"
-				type="checkbox"
-				id="vehicle1"
-				name="vehicle1"
-				value="Bike"
-			/>
-			<label for="vehicle1">
+		<div
+			className={
+				task.isCompleted === false ? 'task task-pending' : 'task task-completed'
+			}
+		>
+			{task.isCompleted ? (
+				<div></div>
```

Comment:
if there is nothing when it isCompleted then rather than ? : u can used &&

Reply:
Yes Changed it